src/app/components/elements/inputs/DobInput.tsx

```diff
@@ -10,11 +10,11 @@ interface DobInputProps {
     submissionAttempted: boolean;
     editingOtherUser?: boolean;
 }
-export const DobInput = ({userToUpdate, setUserToUpdate, submissionAttempted, editingOtherUser}: DobInputProps) => {
+export const DobInput = ({userToUpdate, setUserToUpdate, editingOtherUser}: DobInputProps) => {
     return <RS.FormGroup>
         <RS.Label htmlFor="dob-input">Date of birth</RS.Label>
         <DateInput
-            invalid={isDefined(userToUpdate.dateOfBirth) && !isDobOldEnoughForSite(userToUpdate.dateOfBirth)}
```

Comment:
couple of things here:
the method isDobOldEnoughForSite is more descriptive than the overThirteen - it can be deduced that 13 is the limit but it's now not explicitly stated in code.
is this age validation echoed in the backend?

Reply:
The previous version of the code had a number of age verification variables, including "confirmedOverThirteen", "confirmedOverTen", "confirmedTenToTwelve", "confirmedOldEnoughForSite", then "isDobOldEnoughForSite" was the one that took those, checked whether CS/Physics and applied age restriction accordingly. Since 13 is the only applicable age restriction on CS I simplified it down to the overThirteen version. 

However, probably not much point to change it to something less explicit like isDobOldEnoughForSite, as in the new registration process we won't be gathering DOB anymore. Students will still have a tickbox to say they are over 13, but there won't be any verification happening after the next sprint hopefully.

A quick look over the backend and I can't see any verification happening there, however getting rid of DoB entirely will mean we can get rid of some great stuff like this:

"        // So, DOB is a date in the database.
        // Because Java is Java, and this code goes back to 1996, the date is interpreted as a datetime at midnight
        // in the local timezone. This is obviously insane, but there you go.
        // It works on the servers because they are sensibly set to a UTC timezone. However, it is broken on local
        // machines where the timezone is London time.
        // The front-end expects a timestamp in UTC, so this line uses four deprecated functions to make one. The joy!"